utils/superset/import_template.json

```diff
@@ -0,0 +1,5910 @@
+                "slices": [
+                    {
+                        "__Slice__": {
+                            "cache_timeout": null,
+                            "datasource_name": "bdr.node_summary",
+                            "datasource_type": "table",
+                            "id": 246,
+                            "params": "{\"adhoc_filters\": [], \"all_columns\": [\"node_name\", \"node_group_name\", \"peer_state_name\", \"peer_target_state_name\", \"node_id\", \"interface_connstr\"], \"color_pn\": true, \"conditional_formatting\": [], \"datasource\": \"201__table\", \"extra_form_data\": {}, \"groupby\": [], \"include_search\": false, \"order_by_cols\": [], \"order_desc\": true, \"percent_metrics\": [], \"query_mode\": \"raw\", \"row_limit\": 10000, \"server_page_length\": 10, \"show_cell_bars\": true, \"slice_id\": 220, \"table_timestamp_format\": \"%Y-%m-%d %H:%M:%S\", \"time_grain_sqla\": \"P1D\", \"time_range\": \"No filter\", \"time_range_endpoints\": [\"inclusive\", \"exclusive\"], \"viz_type\": \"table\", \"remote_id\": 246, \"datasource_name\": \"node_summary\", \"schema\": \"bdr\", \"database_name\": \"template_database_name\", \"import_time\": 1657631574}",
```

Comment:
It looks like params are json blobs as strings? Is the file format documeted somewhere?

Reply:
Yes, the strings in the param are JSON blobs and we are loading that in `json.loads()` 
Don't have any docs specified for the file format.